src/pages/dashboard/HomePage.tsx

```diff
@@ -48,9 +48,10 @@ const HomePage = () => {
 
   const currentPath = location.pathname.replace(/\/$/, ''); // Remove trailing slash
 
-  const resultIsSelected = !paramsSearchType || paramsSearchType === SearchTypeValue.Result;
-  const personIsSeleced = paramsSearchType === SearchTypeValue.Person;
-  const projectIsSelected = paramsSearchType === SearchTypeValue.Project;
+  const isOnFilterPage = location.pathname === UrlPathTemplate.Home;
```

Comment:
Er egentlig FilterPage noe? Bedre med `LandingPage`, eller evt. `HomePage`? UrlPathTemplate.Home hinter jo til HomePage. 

Bare pirker litt, så opp til deg hva du synes :) 

```suggestion
  const isOnLandingPage = location.pathname === UrlPathTemplate.Home;
```

Reply:
Greia der er at det er flere ting som er på Home-page :(
Vi vil droppe disse søkene når brukeren er på Avansert søk, eller Rapporter

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/02daab82-f28f-449d-9bf1-2f99a4e9c858)

Reply:
Dette ble enda litt rarere når avansert søk ble flyttet til toppen, siden vår rot-URL (`/`) nå er på menypunkt 2

Reply:
Her skinner min uvitenhet igjennom :) Tenkte ikke over at menyelementet heter "filter". Kjør på 👍